code/__DEFINES/maps.dm

```diff
@@ -44,7 +44,7 @@ Always compile, always use that verb, and always make sure that it works for wha
 #define SPACERUIN_MAP_EDGE_PAD 15
 
 /// Distance from edge to move to another z-level
-#define TRANSITIONEDGE 7
+#define TRANSITIONEDGE 8
```

Comment:
speak on this

Reply:
also should it be like 9 or whatever cause fullscreen

Reply:
Yeah cause uhh we increased the screen width a while ago and never updated this. It doesn't really stand out when you're in space because it's black on black, but it does in icebox.

Also width transition edge is +1, so this is the correct value for the define

<img width="740" height="280" alt="image" src="https://github.com/user-attachments/assets/070eca2c-8931-4f04-893f-efc45fcff81b" />